### ISSUE TITLE
Update Flipper UI to latest

### ISSUE THREAD

**Issue by ryan-mcneil** [department-of-veterans-affairs/va.gov-team#91547]
## Issue Description
When the backend CoP was approached by a VFS team with a feature request, we realized that there are significant updates that have been added to the Flipper UI gem that we had forked years ago and never updated. It's possible that pulling in these new features may satisfy or partially satisfy part of the request.

Notes:
* We should investigate if it's possible to insert the standard views as partials so that we can either receive regular Flipper UI updates, or make it easier to keep it up-to-date in the future.
* This may not be reasonable because there are parts of the views are that are either read-only or not accessible at all based on github teams authorization
---
## Tasks
- [x] Investigate possibility of using standard views from the gem, or if we must copy the views as we did before
- [x] Based on the result from above:
  - [x] Implement a pattern to render standard views within a layout that we can control authorization to certain features
  \- or - 
  - [ ] Copy view (and controller?) code and add our authorization features (similar to current implementation)
- [x] Confirm all authorization mechanics (via github teams) are working as expected
  - [x]  Index and show pages are read only when unauthorized
  - [x]  ~~Views redirect to Index~~ Views redirect to respective pages
  - [x]  Github Button behaves as expected

## Acceptance Criteria
- [x] ~~Feature has been tested (locally and in staging) and deployed to production~~ Will be deployed in #94800 
- [x] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by ryan-mcneil** [comment]
Related PRs for past work:
* Initial Implementation ([part 1](https://github.com/department-of-veterans-affairs/vets-api/pull/3188), [part 2](https://github.com/department-of-veterans-affairs/vets-api/pull/3189), part 3
*  [UI customization](https://github.com/department-of-veterans-affairs/vets-api/pull/3375)

**Comment by ryan-mcneil** [comment]
Update:

Slow to get off the ground thanks to the slow load times that we pull into this ticket to investigate separately.

Work is now almost complete, pending updates to specs + feedback. PR [here](https://github.com/department-of-veterans-affairs/vets-api/pull/18694)

**Comment by ryan-mcneil** [comment]
As some unexpected scope was discovered that needs to be done before deploying, a separate ticket was created: #94800 . This will handle an unexpected bug that lead to finding gaps in work related to Actors/Groups/%'s that weren't expected here, and were missed in the last iteration 9 months ago. This ticket will be closed as this work is finished.
